sql/hive/src/test/scala/org/apache/spark/sql/hive/InsertSuite.scala

```diff
@@ -581,6 +581,38 @@ class InsertSuite extends QueryTest with TestHiveSingleton with BeforeAndAfter
     }
   }
 
+  test("insert overwrite to not exists dir from hive metastore table") {
+    withTempDir { dir =>
+      val path = dir.toURI.getPath
+      val notExistsPath = s"${path}/src/result"
+
+      sql(s"INSERT OVERWRITE LOCAL DIRECTORY '${notExistsPath}' SELECT * FROM src where key < 10")
```

Comment:
nit: `${notExistsPath}` -> `$notExistsPath `

Reply:
OK, I have adjust it.